est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
```

Comment:
@dalifreire lets call this "Implement" instead of "Design" so that the next guy reading this understands that this refers to a coding task properly.